client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -166,9 +178,12 @@ class UpgradeNudge extends React.Component {
 export default connect(
 	( state, ownProps ) => {
 		const siteId = getSelectedSiteId( state );
+		const site = getSelectedSite( state );
+
 		return {
-			site: getSelectedSite( state ),
+			site,
 			planHasFeature: hasFeature( state, siteId, ownProps.feature ),
+			canUserCanManageSite: canCurrentUser( state, siteId, 'manage_options' ),
```

Comment:
minor, but maybe `userCanManageSite`

Reply:
what was I thinking? That's a super copy-paste snafu. Anyways, I've checked the uses of `canCurrentUser`, and usually the prop/variable name attached to it starts with `can*`, so I've changed it on 51d1672efb856cb5e23998a52eb12bec666b302c to `canManageSite`.